workers/unity/Packages/io.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -111,16 +109,6 @@ public void SendLogMessage(LogLevel logLevel, string message, string loggerName,
             MessagesToSend.AddLogMessage(new LogMessageToSend(message, loggerName, logLevel, entityId?.Id));
         }
 
-        /// <summary>
-        ///     Gets the value for a given worker flag.
-        /// </summary>
-        /// <param name="key">The key of the worker flag.</param>
-        /// <returns>The value of the flag, if it exists, null otherwise.</returns>
-        public string GetWorkerFlag(string key)
-        {
-            return View.GetWorkerFlag(key);
-        }
```

Comment:
Given this was exposed here before.. can we keep the worker flags in here and not add another breaking change?